prov/efa/src/efa.h

```diff
@@ -356,6 +355,8 @@ struct efa_reverse_av_key {
 	uint32_t connid;
 };
 
+#define EFA_DGRAM_CONNID (0x0)
```

Comment:
Didn't realize that `EFA_CONNID_NOTAVAIL` was going to get dropped, anyway the comment in the last commit still applies.